package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
These configs in the root are not necessary, only need to add them in the `react/package.json`.